languages/rust/oso/Cargo.toml

```diff
@@ -42,6 +42,7 @@ rustyline-derive = { version = "0.3.1", optional = true }
 lazy_static = "1.4.0"
 
 uuid-06 = { package = "uuid", version = "0.6.5", optional = true }
+uuid-07 = { package = "uuid", version = ">=0.7.0, <0.9.0", optional = true }
```

Comment:
I'm not sure this will be able to support both version simultaneously?

Reply:
Yes this will let the user decide which version use, the only requirement is that the version of `uuid` specified in the user's `Cargo.toml` is the same of the one which depends `oso`, specified in the user's `Cargo.lock`.

Reply:
If that's the case, do we even need a separate one for uuid 6 and 7/8?

Reply:
I think that this is the only way if you would retain the compability with uuid 0.6, but I would happy to be proved wrong.

Reply:
So as far as I can understand it. It would be fine for us to make the `uuid` feature `"*"`. And whatever version people are using it would be fine. However, if anyone is using multiple versions of uuid in one project, then PolarClass would only be implemented for one.